Dockerfile

```
@@ -9,39 +9,43 @@ LABEL org.label-schema.build-date=$BUILD_DATE \
   org.label-schema.name="docker-phpcs-drupal" \
   org.label-schema.description="PHP codesniffer for Drupal - phpcs & phpcbf" \
   org.label-schema.vcs-url="https://github.com/skilld-labs/docker-phpcs-drupal" \
-  maintainer="Andriy Yun <andriy.yun@gmail.com>, Andy Postnikov <apostnikov@gmail.com>"
+  maintainer="Andriy Yun , Andy Postnikov "
 
 RUN set -e \
   && apk add --no-cache \
   curl \
   git \
   patch \
-  php7 \
-  php7-apcu \
-  php7-ctype \
-  php7-json \
-  php7-mbstring \
-  php7-opcache \
-  php7-openssl \
-  php7-phar \
-  php7-simplexml \
-  php7-tokenizer \
-  php7-xmlwriter \
-  php7-zlib \
-  && curl -sS https://getcomposer.org/installer | php -- --filename=composer --install-dir=/usr/bin \
+  php83 \
+  php83-apcu \
+  php83-ctype \
+  php83-json \
+  php83-mbstring \
+  php83-opcache \
+  php83-openssl \
+  php83-phar \
+  php83-simplexml \
+  php83-tokenizer \
+  php83-xmlwriter \
+  php83-zlib;
+
+RUN ln -s $(which php8) /usr/local/bin/php;
+
+RUN curl -sS https://getcomposer.org/installer | php -- --filename=composer --install-dir=/usr/bin \
```

```suggestion
  && curl -sS https://getcomposer.org/installer | php -- --filename=composer --install-dir=/usr/bin \
```
please minimize number of layers

---

Also not clear about naming - we should start using tags containing coder version I guess